functions/dataManger/priceDataManager.ts

```diff
@@ -0,0 +1,63 @@
+import { Game } from "../models/game";
+import { GameError } from "../error/gameErrorHandler";
+import { PriceData } from "../models/priceData";
+import * as cheerio from "cheerio";
+let axios = require('axios').default;
+
+export async function getPriceData(game: Game) : Promise<PriceData> {
+  if (!game.desiredPrice)
+    throw new GameError("No price associated with this game.", 400);
+  let url = `https://www.pricecharting.com/search-products?type=prices&q=${game.gameName}`;    
```

Comment:
Might be time to start playing with external configuration sources. Either DDB or SystemsParameterStore